.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  check-lint:
+    strategy:
+      matrix:
+        include:
+          - name: flake8
+            tox-env: flake8
+          - name: pyupgrade
+            tox-env: pyupgrade
+
+    name: Check ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install dependencies
+        run: |
+          set -xeu
+          python -VV
```

Comment:
why `-VV`? how is it different/better than `python --version`?

Reply:
(I'd tend to favor `--version` too)